cluster_create_cli.adoc

```diff
@@ -0,0 +1,56 @@
+[#create-a-cluster]
+= Create a cluster
+
+Learn how to create OpenShift Container Platform clusters with Hive, the multicluster engine for Kubernetes component for provisioning clusters. The Hive Controller provisions the clusters that you create with multicluster engine for Kubernetes.
+
+* <<create-a-cluster-with-cluster-deployment,Create a cluster with ClusterDeployment>>
+* <<create-a-cluster-with-clusterpool,Create a cluster with cluster pool>>
+
+[#create-a-cluster-with-clusterdeployment]
+== Create a cluster with ClusterDeployment
+
+Learn how to create a cluster directly for management.
```

Comment:
The word "directly" seems odd here.  I think its a reference to the idea of creating a single "standalone" (not in a pool) cluster vs. the cluster pool stuff that follows.  Also, the "...for management" seems odd.  I don't think a customer would be creating a cluster primarily because they want to manage it, they would be creating it for some workload purpose and the need to manage it is a "side effect".

I suggest just saying "Learn how to create an individual cluster."

Reply:
copied to issue